src/EncryptedSessionsZF2/Factory/EncryptedSessionSaveHandlerFactory.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace EncryptedSessionsZF2\Factory;
+
+use Zend\ServiceManager\FactoryInterface;
+use Zend\ServiceManager\ServiceLocatorInterface;
+
+use Zend\Session\SessionManager;
+use Zend\Session\Exception\RuntimeException;
+use Zend\Session\SaveHandler\SaveHandlerInterface;
+
+use EncryptedSessionsZF2\Session\SaveHandler\EncryptedSessionSaveHandler;
+use EncryptedSessionsZF2\Session\SaveHandler\NullSessionSaveHandler;
+
+/**
+ * Factory used to instantiate an encrypted session save handler.
+ */
+class EncryptedSessionSaveHandlerFactory implements FactoryInterface
+{
+    /**
+     * {@inheritDoc}
+     * @return EncryptedSessionSaveHandler
+     * @throws RuntimeException if configuration is missing.
+     */
+    public function createService(ServiceLocatorInterface $serviceManager)
+    {
+        $config = $serviceManager->get('Config');
+
+        if (!isset($config['encrypted_sessions_zf2'])) {
+            throw new RuntimeException('Have you copied
```

Comment:
This string could possibly be broken up and concatenated to remove line breaks from the output.